lib/mcb/commands/sites/geocode.rb

```diff
@@ -0,0 +1,31 @@
+name "geocode"
+summary "Batch geocode Sites"
+usage "geocode [options] [<id>...]"
+option :s, "sleep", "time to sleep between each request", argument: :optional, default: 0.25, transform: method(:Float)
+option :b, "batch_size", "batch size", argument: :optional, default: 100, transform: method(:Integer)
+
+instance_eval(&MCB.remote_connect_options)
+
+require "geocoder"
+
+# TODO: Once Site codes have been changed to three digit values
```

Comment:
Currently site codes are one character long and we have 17000 sites in prod, hence lots of duplicate codes.